### ISSUE TITLE
[FEATURE] Save the output file somewhere logical

### ISSUE THREAD

**Issue by allisonletts** [toddhalfpenny/sfflowvisualiser-vscode#10]
The .md file is not created inside the project folder, so I get a VS error when I attempt to save. I'd love to be able to keep these inside my repo :smile:
Options:
1. Create a new folder called flow-vis or something at the project root
2. Just put the files in the project root
3. Give a settting to determine where the visualizations should be stored, relative to the project root.

**Comment by toddhalfpenny** [comment]
Ooooh... yes, good point.
Do you think the option to save the whole markdown file or just the diagram part (the mermaid lines) would be best? Or the option for both, @allisonletts ?

**Comment by allisonletts** [comment]
The whole markdown file would be best, I think, but an option would also be nice.

**Comment by toddhalfpenny** [comment]
Nice... thanks.
I'll put this on the list for the next version.

**Comment by JodieM** [comment]
+1 from me. Is it still an .md file with Webview? I can't even see how to save or download the whole file now. I would love to save the file either as markdown or html in my documentation.

**Comment by allisonletts** [comment]
Yeah - being able to save the file is important to me as well, or even just the part that isn't the diagram. At the moment I'm copy/pasting or just rewriting the top half of the page, and then saving the image separately.
